### DISCUSSION TITLE
Menu related

### DISCUSSION THREAD

**Discussion by skhushbu2798** [ng-matero/ng-matero#552] [Q&A]
1. how to add own JSON string to menu?
2. How to call our own API for Login Dynamically?

**Comment by nzbin** [comment]
Please check the login service in `core/authentication`. If you install the app with `ng add`, you should remove the `fake-login.service` first.

1. Modify the menu API to get your own data.
https://github.com/ng-matero/ng-matero/blob/97e3b3d66393948d3a28a8e9fa7f8e1f619bd368/src/app/core/authentication/login.service.ts#L29-L31
The menu data should follow the types below. You can also modify the `menu.json` to get static menu data.
https://github.com/ng-matero/ng-matero/blob/97e3b3d66393948d3a28a8e9fa7f8e1f619bd368/src/assets/data/menu.json#L1-L11

2. Modify the login API.
https://github.com/ng-matero/ng-matero/blob/97e3b3d66393948d3a28a8e9fa7f8e1f619bd368/src/app/core/authentication/login.service.ts#L13-L15
